Python/compile.c

```diff
@@ -55,11 +55,14 @@ struct compiler_unit {
     PyObject *u_private;            /* for private name mangling */
     PyObject *u_static_attributes;  /* for class: attributes accessed via self.X */
     PyObject *u_deferred_annotations; /* AnnAssign nodes deferred to the end of compilation */
+    PyObject *u_conditional_annotation_indices;  /* indices of annotations that are conditionally executed (or -1) */
+    int u_next_conditional_annotation_index;  /* index of the next conditional annotation */
```

Comment:
Shouldn't it be a long at least? I think we're storing long indices?

Reply:
Got to handle those classes with 2 billion fields :P